packages/server-core/src/projects/project/project-helper.ts

```diff
@@ -112,73 +112,94 @@ export const updateBuilder = async (
 
   if (helmSettings && helmSettings.builder && helmSettings.builder.length > 0)
     await execAsync(
-      `helm upgrade --reuse-values --version ${helmSettings.builder} --set builder.image.tag=${tag} ${builderDeploymentName} etherealengine/etherealengine-builder`
+      `kubectl delete job --ignore-not-found=true ${config.server.releaseName}-builder-etherealengine-builder && helm upgrade --reuse-values --version ${helmSettings.builder} --set builder.image.tag=${tag} ${builderDeploymentName} etherealengine/etherealengine-builder`
```

Comment:
Since our job name is coming from `etherealengine-builder/templates/_helpers.tpl` and there if we dont have fullname override defined then `{{- printf "%s-%s" .Release.Name .Values.builder.name | trunc 63 | trimSuffix "-" -}}` will be used.

Maybe if easily possibly, this and below usage in else block can be made some what dynamic using a `k8BatchClient.listNamespacedJob()` version of:
https://github.com/EtherealEngine/etherealengine/blob/9b477608c821efe42737f6b974f5125a845b24ae/packages/server-core/src/projects/project/project.service.ts#L181

Reply:
Good suggestion, I've changed the code to do this.